src/terra-dev-site/doc/how-to.2/show-modal-content.doc.mdx

```diff
@@ -0,0 +1,57 @@
+# How To Show Modal Content
+
+The [ModalManager](/components/terra-application/reference/components/modal-manager) should be used to manage modal content presentation. 
+
+```jsx
+import React from 'react';
+import ApplicationBase from 'terra-application/lib/application-base';
+import ModalManager from 'terra-application/lib/modal-manager';
+
+import ComponentWithModal from './ComponentWithModal';
+
+const ExampleApp = () => (
+  <ApplicationBase>
```

Comment:
Should we write these examples with the assumptions someone is already in an app that has already implemented ApplicationBase? Adding it here seems like a a copy-paste hazard where consumers will be implementing multiple layers of `ApplicationBase`

Reply:
Or should this how to for the base application level?

Reply:
I like how the example separates the application setup from the component doing the discosure and I think all it needs is a little more description about 'this is how the app is setup and you may not need to render a modal manager if you're a child of a component that already rendered it'...

Reply:
I'll elaborate